### ISSUE TITLE
Install on Rhino / Ubuntu ?

### ISSUE THREAD

**Issue by iconoclasthero** [zsh-users/zsh-history-substring-search#166]
tl;dr: simplest option for installing this on rhino linux/ubuntu devel having zero prior knowledge of zfs?

I briefly ran Manjaro recently and really want to try to replicate the terminal experience there.  Apparently this is a dependency for manjaro-zsh-config so it's necessary for that.

I use bash and guake on ubuntu (now rhino) and have no idea what any of those install options are.  I started looking at them and quickly got overwhelmed with options I don't really want and really do not want to vet just to try this.  ([The Paradox of Choice](https://en.wikipedia.org/wiki/The_Paradox_of_Choice) I suppose)
